packages/account/src/Components/poi/poi-form-on-signup/poi-form-on-signup.tsx

```diff
@@ -12,14 +24,15 @@ export const ProofOfIdentityFormOnSignup = ({
     value,
     residence_list,
     citizen,
-}) => {
-    const citizen_data = residence_list.find(residence => residence.value === citizen);
+}: TProofOfIdentityFormOnSignup) => {
+    const citizen_data: FormikValues =
```

Comment:
Do we need to define type for this constant? I guess you can remove it

Reply:
Removed the types 🙏